windows_containers/windows-containers-release-notes-7-x.adoc

```diff
@@ -28,6 +28,23 @@ For more information, see the Red Hat OpenShift Container Platform Life Cycle Po
 If you do not have this additional Red Hat subscription, you can use the Community Windows Machine Config Operator, a distribution that lacks official support.
 endif::openshift-origin[]
 
+[id="wmco-7-2-1"]
+== Release notes for Red Hat Windows Machine Config Operator 7.2.1
+
+This release of the WMCO provides new features and bug fixes for running Windows compute nodes in an {product-title} cluster. The components of the WMCO 7.2.1 were released in link:https://access.redhat.com/errata/RHSA-{TBD}[RHSA-{TDB}].
+
+[id="wmco-7-2-1-bug-fixes"]
+=== Bug fixes
+
+// Also in 10.15.0
+* Previously, the WMCO did not properly wait for Windows VMs to finish rebooting. This led to occasional timing issues where the WMCO would attempt to interact with a node that was in the middle of a reboot, causing WMCO to log an error and restart node configuration. Now, the WMCO waits for the instance to completely reboot. (link:https://issues.redhat.com/browse/OCPBUGS-23036[*OCPBUGS-23036*])
+
+// Also in 10.15.0
+* Previously, the WMCO configuration was missing the `DeleteEmptyDirData: true` field, which is required for draining nodes that have emptyDir volumes attached. As a consequence, customers that had nodes with emptyDir data volumes would see the following error in the logs: `cannot delete Pods with local storage`. With this fix, the `DeleteEmptyDirData: true` field was added to the node drain helper struct in the WMCO. As a result, customers are able to drain nodes with emptyDir volumes attached. (link:https://issues.redhat.com/browse/OCPBUGS-23081[*OCPBUGS-23081*])
```

Comment:
Question - do you know if the phrase "emptyDir" shows up in any code or file, and if that's what's specifically being referred to here? If so, I think it should be in literals:
```suggestion
* Previously, the WMCO configuration was missing the `DeleteEmptyDirData: true` field, which is required for draining nodes that have `emptyDir` volumes attached. As a consequence, customers that had nodes with `emptyDir` data volumes would see the following error in the logs: `cannot delete Pods with local storage`. With this fix, the `DeleteEmptyDirData: true` field was added to the node drain helper struct in the WMCO. As a result, customers are able to drain nodes with `emptyDir` volumes attached. (link:https://issues.redhat.com/browse/OCPBUGS-23081[*OCPBUGS-23081*])
```